cumulus/scripts/bridges_common.sh

```diff
@@ -19,9 +19,9 @@ function ensure_relayer() {
 }
 
 function ensure_polkadot_js_api() {
-    if ! which polkadot-js-api &> /dev/null; then
+    if ! which call_polkadot_js_api &> /dev/null; then
         echo ''
-        echo 'Required command `polkadot-js-api` not in PATH, please, install, e.g.:'
+        echo 'Required command `call_polkadot_js_api` not in PATH, please, install, e.g.:'
```

Comment:
just a nit, `call_polkadot_js_api` is a bash function which wraps `polkadot-js-api` command, so I think here should be really check for `polkadot-js-api`, right?

Reply:
Yes, you're right! Thank you!